lib/kaui.rb

```diff
@@ -201,6 +201,10 @@ def self.config
       layout: layout || 'kaui/layouts/kaui_application'
     }
   end
+
+  def self.additional_headers_partial
```

Comment:
You could make this a `mattr_accessor` so that we can simply to: `Kaui.additional_headers_partial = 'layouts/additional_header'` in killbill-admin-ui-standalone